helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -87,3 +95,23 @@ monitoring:
         - "0.0.0.0/0"
       ports:
         - 443
+dex:
+  enabled: true
+
+falco:
+  enabled: true
+
+coredns:
+  enabled: true
+  externalDns:
+    ips:
+      - "172.16.0.2/32"
+      - "172.16.0.3/32"
+
+nodeLocalDns:
+  enabled: true
+
+dnsAutoscaler:
+  enabled: true
+calicoAccountant:
+  enabled: true
```

Comment:
Please expose this to the end config via the values file as well. And I assume that the ips here are from your testing.

Reply:
done